packages/lib/src/settings/options/itemShadowBlur.js

```diff
@@ -3,6 +3,8 @@ import { default as GALLERY_CONSTS } from '../../common/constants';
 
 export default {
   title: 'Item Shadow Blur',
+  isRelevantDescription:
+    'Set a Vertical gallery ("Scroll Direction" as "Vertical") and set "Crop Images" to "false" or set "Crop Type" to anything but "Fit" and set "Choose info layout" to "Attached Background" or set "Texts Placement" to "Show On Hover" and set "Enable Item Shadow" to "true"',
```

Comment:
First set a Vertical gallery ("Scroll Direction" as "Vertical").
Then set "Crop Images" to "false" or set "Crop Type" to anything but "Fit".
Then set "Choose info layout" to "Attached Background" or set "Texts Placement" to "Show On Hover".
Then set "Enable Item Shadow" to "true".